gunrock/app/hits/hits_app.cuh

```diff
@@ -205,21 +201,21 @@ double gunrock_hits(
 }
```

Comment:
What if we have number of runs larger than 1 here? Will `gunrock_hits` capture all the runtime?

Reply:
No other apps have a parameter to specify a number of runs greater than 1. num_iter can be used to specify how many HITS iterations to perform, since this is an iterative algorithm.

Reply:
Other apps have the option of changing the number of runs to multiple. And in gunrock_xx they are accumulating the total time ran by `num_runs`.

Reply:
That's not particularly important, because you could always have a loop outside the run and do the cpu_timer accumulation yourself at the c call level.